spec/integration_spec.rb

```diff
@@ -325,7 +325,7 @@ def parse_html(html)
       expect(result).to have_styling('color' => 'green').at_selector('.colorful')
     end
 
-    it "should put non-inlineable media queries in the head" do
+    it 'puts non-inlineable media queries in the head' do
```

Comment:
Please change these quotes back and make sure all other examples you added use double quotes too (unless single quotes keeps you from having to escape a double quote).

```ruby
it "foo"
it 'says "foo" to me'
```